fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/jobs/updateloanarrearsageing/UpdateLoanArrearsAgeingTasklet.java

```diff
@@ -84,18 +98,25 @@ public RepeatStatus execute(StepContribution contribution, ChunkContext chunkCon
         }
 
         log.debug("{}: Records affected by updateLoanArrearsAgeingDetails: {}", ThreadLocalContextUtil.getTenant().getName(), result);
-        return RepeatStatus.FINISHED;
     }
 
-    private List<String> updateLoanArrearsAgeingDetailsWithOriginalSchedule() {
+    private List<String> updateLoanArrearsAgeingDetailsWithOriginalSchedule(String loanIdsForUpdate) {
         List<String> insertStatement = new ArrayList<>();
 
         final StringBuilder loanIdentifier = new StringBuilder();
         loanIdentifier.append("select ml.id as loanId FROM m_loan ml  ");
```

Comment:
Ruchi, wild idea but don't we wanna use String blocks (introduced in Java15) to make this more readable? (`"""`)